ammonite/httpMethods.sc

```diff
@@ -126,6 +126,35 @@ object Test {
                  ).get,
        keyChain = List(cosyKey,localKey)
    )
+   
+   //----------------------------------------------------------------------------------------------
+
+   def deleteLocalKey = http.run(
+     req      = DELETE[`text/turtle`](
+                  resource = Uri("https://localhost:8443/2013/")
+                )
+   )
```

Comment:
why do you want to delete the whole root container? Would it not be better to delete a file that was previously created perhaps?